core/players-handbook-2024/archetypes/bard-dance.xml

```diff
@@ -0,0 +1,91 @@
+			<div element="ID_WOTC_PHB24_ARCHETYPE_FEATURE_BARD_DANCE_DAZZLING_FOOTWORK" />
+			<div element="ID_WOTC_PHB24_ARCHETYPE_FEATURE_BARD_DANCE_INSPIRING_MOVEMENT" />
+			<div element="ID_WOTC_PHB24_ARCHETYPE_FEATURE_BARD_DANCE_TANDEM_FOOTWORK" />
+			<div element="ID_WOTC_PHB24_ARCHETYPE_FEATURE_BARD_DANCE_LEADING_EVASION" />
+		</description>
+		<sheet display="false" />
+		<rules>
+			<grant type="Archetype Feature" id="ID_WOTC_PHB24_ARCHETYPE_FEATURE_BARD_DANCE_DAZZLING_FOOTWORK" level="3" />
+			<grant type="Archetype Feature" id="ID_WOTC_PHB24_ARCHETYPE_FEATURE_BARD_DANCE_INSPIRING_MOVEMENT" level="6" />
+			<grant type="Archetype Feature" id="ID_WOTC_PHB24_ARCHETYPE_FEATURE_BARD_DANCE_TANDEM_FOOTWORK" level="6" />
+			<grant type="Archetype Feature" id="ID_WOTC_PHB24_ARCHETYPE_FEATURE_BARD_DANCE_LEADING_EVASION" level="14" />
+		</rules>
+	</element>
+
+	<element name="Level 3: Dazzling Footwork" type="Archetype Feature" source="Player’s Handbook 2024" id="ID_WOTC_PHB24_ARCHETYPE_FEATURE_BARD_DANCE_DAZZLING_FOOTWORK">
+		<requirements>!ID_INTERNAL_PHB24_FEATURE_REPLACEMENT_BARD_DANCE_DAZZLING_FOOTWORK</requirements>
+		<description>
+			<p>While you aren’t wearing armor or wielding a Shield, you gain the following benefits:</p>
+			<p class="indent"><em><strong>Dance Virtuoso.</strong></em> You have Advantage on any Charisma (Performance) check you make that involves you dancing.</p>
+			<p class="indent"><em><strong>Unarmored Defense.</strong></em> Your base Armor Class equals 10 plus your Dexterity and Charisma modifiers.</p>
+			<p class="indent"><em><strong>Agile Strikes.</strong></em> When you expend a use of your Bardic Inspiration as part of an action, a Bonus Action, or a Reaction, you can make one Unarmed Strike as part of that action, Bonus Action, or Reaction.</p>
+			<p class="indent"><em><strong>Bardic Damage.</strong></em> You can use Dexterity instead of Strength for the attack rolls of your Unarmed Strikes. When you deal damage with an Unarmed Strike, you can deal Bludgeoning damage equal to a roll of your Bardic Inspiration die plus your Dexterity modifier, instead of the strike’s normal damage. This roll doesn’t expend the die.</p>
+		</description>
+		<sheet alt="Dazzling Footwork">
+			<description>While you aren’t wearing armor or wielding a Shield, your base AC is {{ac:dazzling footwork}}, and you have Advantage on any Charisma (Performance) check you make that involves you dancing. &#13;
+			Additionally, you can make one Unarmed Strike when you expend a use of your Bardic Inspiration, which deals 1d{{bardic-inspiration:dice}}+{{dazzling footwork:damage}} bludgeoning damage, and you can use either Strength or Dexterity for the attack rolls.</description>
```

Comment:
```suggestion
			<description>While you aren’t wearing armor or wielding a Shield, your base AC is {{ac:dazzling footwork}}. You have Advantage on any Charisma (Performance) check you make that involves you dancing. &#13;
			Additionally, you can use Dexterity instead of Strength for your Unarmed Strikes, which can deal 1d{{bardic-inspiration:dice}}+{{dexterity:modifier}} bludgeoning damage instead of the normal amount. Whenever you expend a use of your Bardic Inspiration, you can make one Unarmed Strike as part of that action, Bonus Action or Reaction.</description>
```